### DISCUSSION TITLE
Why RDKit fails to generate conformers for bridged bicyclic compounds (e.g., Scopolamine, Dolasetron) ?

### DISCUSSION THREAD

**Discussion by xli7654321** [rdkit/rdkit#7954] [Q&A] [answered]
My RDKit version is `2023.9.5`
After adding hydrogen to mol, I tried several methods but could not get the conformer. I wonder if there is another solution.
```python
from rdkit import Chem
from rdkit.Chem import AllChem
from rdkit.Chem.Draw import IPythonConsole
IPythonConsole.ipython_3d = True

# Dolasetron
smi = 'O=C(O[C@@H]1C[C@@H]2C[C@H]3C[C@H](C1)N2CC3=O)c1c[nH]c2ccccc12'
mol = Chem.MolFromSmiles(smi)
mol = Chem.AddHs(mol)

# attempt 1
AllChem.EmbedMolecule(mol, randomSeed=42)

# attempt 2
AllChem.EmbedMolecule(mol, randomSeed=42, useRandomCoords=True)

# attempt 3
AllChem.EmbedMolecule(mol, randomSeed=42, maxAttempts=10000)
```

All three attempts return -1, i.e., no conformer could be generated.

AND I noticed a similar question mentioned at #7179, using `enforceChirality=False` indeed solved the problem, but I still do not know exactly what has changed.

**Comment by greglandrum** [comment, marked as answer]
This is most likely happening because the stereochemistry you are providing is actually wrong. With complex ring systems like this, it is often the case that only one stereoisomer is actually realizable without doing something like putting an H into the middle of a fused ring system.

It's difficult to be sure here, but I guess that's probably what is happening. If I use the SMILES for Dolasetron from wikipedia everything embeds fine:
```
In [11]: m = Chem.AddHs(Chem.MolFromSmiles('[H]C35C[C@@]4([H])CC(OC(=O)c1c[nH]c
    ...: 2ccccc12)C[C@@]([H])(C3)N4CC5=O'))

In [12]: rdDistGeom.EmbedMolecule(m,randomSeed=42)
Out[12]: 0
```
If I use the ChEMBL SMILES (which looks like what you have), I see the same problem you do. I guess that there's a problem with the ChEMBL SMILES (FWIW: I also cannot generate conformers after reading in the ChEMBL Mol block) , but these things are time consuming to track down.

**Reply by xli7654321** [reply]
Thanks for your answer. Yes, this is indeed caused by wrong stereochemistry. When I corrected it, the error no longer occurred. However, one issue is that I tried the Dolasetron's SMILES from well-known databases such as DrugBank, ChEMBL, and PubChem, and the stereochemistry in all of them seems to be wrong. This might be worth noting.

**Comment by ptosco** [comment]
Greg is right, this is similar to #7976. To satisfy the stereochemical constraints encoded by your SMILES
![image](https://github.com/user-attachments/assets/6077f7ba-39db-499b-a265-e7ed9922fdf0)
the bridge between atoms 7 and 9 should cross the plane of the bicyclic system, giving rise to a geometry which is too strained to be feasible.
